### PR TITLE
Cleanup live components

### PR DESCRIPTION
**Pull request by philschanely**
- [x] Fix default color of session title
- [x] Fix positioning
- [x] Fix hover effects
- [x] Implement better prototyping with partials

### CODE REVIEW

File: app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss [hunk 1]
```diff
@@ -6,44 +6,68 @@
...
-$-sage-live-stream-control-start-color: sage-color(white);
-$-sage-live-stream-control-stop-background-color: sage-color(red, 300);
-$-sage-live-stream-control-stop-color: sage-color(white);
+$-live-stream-control-size: rem(40px);
+$-live-stream-control-background-color: sage-color(white);
+$-live-stream-control-color: sage-color(charcoal, 400);
+$-live-stream-control-flat-background-color: transparent;
+$-live-stream-control-flat-color: sage-color(grey, 100);
+$-live-stream-control-hover-scale: (rem(2px) + $-live-stream-control-size) / $-live-stream-control-size;
+$-live-stream-control-icon-size: rem(16px);
```
Comment (teenwolfblitzer) [inline app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss:15]: I think we can use `sage-font-size(md)` here

File: app/assets/stylesheets/sage/system/patterns/objects/_live_stream_footer.scss [hunk 2]
```diff
@@ -16,8 +16,22 @@
   height: $sage-live-stream-header-footer-height;
   padding: 0 sage-spacing(md);
 
-  .sage-live-stream-wrapper--awake & {
+  &::after {
+    content: "";
+    z-index: -1;
```
Comment (teenwolfblitzer) [inline app/assets/stylesheets/sage/system/patterns/objects/_live_stream_footer.scss]: Can use `sage-z-index(negative)` here

File: app/views/sage/examples/objects/live_stream_control/_markup.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<button class="sage-live-stream-control <%= modifiers %>">
+  <i class="sage-live-stream-control__icon sage-icon-<%= icon %>"></i>
```
Comment (teenwolfblitzer) [inline app/views/sage/examples/objects/live_stream_control/_markup.html.erb]: Voiceover is reading the pseudo-element content for each icon in Safari (displayed as a question mark). I'm wondering if we'll need to either provide a descriptive `aria-label` for each icon, or more likely add `aria-hidden="true"`.

**Review by teenwolfblitzer — COMMENTED** [review]
Looking great, have a question about the icons before ✅

File: app/views/sage/examples/objects/live_stream_control/_preview.html.erb [hunk 4]
```diff
@@ -1,34 +1,46 @@
 <div class="sage-live-stream-demo-wrapper">
-  <button class="sage-live-stream-control">
-    <i class="sage-live-stream-control__icon sage-icon-microphone"></i>
-    <span class="visually-hidden">Button label</span>
-  </button>
+  <%= render "sage/examples/objects/live_stream_control/markup",
```
Comment (teenwolfblitzer) [inline app/views/sage/examples/objects/live_stream_control/_preview.html.erb:2]: 🏅

File: app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss [hunk 5]
```diff
@@ -6,44 +6,68 @@
...
+
+    @include position-full(-$-live-stream-control-ring-offset);
+  }
+  
   &:hover {
-    transform: scale($-sage-live-stream-control-hover-scale);
+    transform: scale($-live-stream-control-hover-scale);
+  }
+
+  &:focus {
```
Comment (teenwolfblitzer) [inline app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss:58]: One more thing: if there aren't `:active` states defined in the design, could it be combined here? In Safari and Firefox, the `:focus` state will not apply with mouse-driven clicks, so the outline effect only appears when using the keyboard to navigate, unlike Chrome which displays both.

Comment (philschanely) [reply app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss:58]: Good catch. Will do.

File: app/views/sage/examples/objects/live_stream_control/_markup.html.erb (hunk 3 above)
Comment (philschanely) [reply app/views/sage/examples/objects/live_stream_control/_markup.html.erb]: Aria-hidden seems like a good option for most uses; I'll look to that in React as well. I realize though that where I'm often adding a visually-hidden label text I could instead use aria-label. Great catch!

File: app/assets/stylesheets/sage/system/patterns/objects/_live_stream_footer.scss (hunk 2 above)
Comment (philschanely) [reply app/assets/stylesheets/sage/system/patterns/objects/_live_stream_footer.scss]: Habit. haha

File: app/views/sage/examples/objects/live_stream_control/_markup.html.erb (hunk 3 above)
Comment (philschanely) [reply app/views/sage/examples/objects/live_stream_control/_markup.html.erb]: I will put in a new ticket to review this throughout the system and update documentation. Aria-label is a clear better solution to the visually-hidden bit I had in place.

File: app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss (hunk 1 above)
Comment (philschanely) [reply app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss:15]: Hey good point; I think we need to talk more about icon sizing in general. In this case I do want a physical size on the icon container and the default font size applied.

File: app/views/sage/examples/objects/live_stream_control/_markup.html.erb (hunk 3 above)
Comment (philschanely) [reply app/views/sage/examples/objects/live_stream_control/_markup.html.erb]: See #156

File: app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss (hunk 1 above)
Comment (philschanely) [reply app/assets/stylesheets/sage/system/patterns/objects/_live_stream_control.scss:15]: See initiated convo here https://kajabi.slack.com/archives/CMNKRU07Q/p1586533053001900

**Review by teenwolfblitzer — APPROVED** [review]
👍🔥LGTM!

File: app/assets/stylesheets/sage/system/patterns/objects/_live_avatar.scss [hunk 6]
```diff
@@ -77,7 +77,7 @@ $-live-avatar-ring-colors: (
 
 // Set up color variations
 @each $color in $-live-avatar-ring-colors {
-  .sage-live-avatar--#{$color} {
+  .sage-live-avatar--#{"" + $color} {
```
Comment (AndrwM) [inline app/assets/stylesheets/sage/system/patterns/objects/_live_avatar.scss:80]: I'm a bit confused about why there's an empty string here. Would you mind explaining?

File: app/assets/stylesheets/sage/system/patterns/objects/_live_stream_footer.scss [hunk 7]
```diff
@@ -16,8 +16,22 @@
...
+  &::after {
+    content: "";
+    z-index: sage-z-index(negative);
+    position: absolute;
     background: linear-gradient(0deg, $sage-live-stream-wrap-gradient-start, $sage-live-stream-wrap-gradient-end);
+    transition: $sage-transition;
+    transition-property: opacity;
+    opacity: 0;
+
+    @include position-full;
```
Comment (AndrwM) [inline app/assets/stylesheets/sage/system/patterns/objects/_live_stream_footer.scss:28]: Cool mixin 😎

**Review by AndrwM — APPROVED** [review]
LGTM +1 !

**Pull request merged**